lib/online_order.rb

```diff
@@ -0,0 +1,53 @@
+require 'awesome_print'
+require 'pry'
+require 'csv'
+
+FILENAME = "support/online_orders.csv"
+
+module Grocery
+  class OnlineOrder < Order
+    # Inherit behavior from order class
+    attr_accessor :status, :customer_id
+
+    def initialize(id, products, status = :pending)
+      super(id, products)
+      @customer_id = customer_id
+      @Status = status
+      # if status == nil
+      #   status = :pending
+      # else
+      #   @status = status.to_sym
+      # end
+    end
+    # same as order except will add $10 shipping fee
+    def total
+      super()
+      return sum += 10
```

Comment:
In order for this to work, you'll need to keep track of the value you get from `super`. Something like:

```ruby
def total
  sum = super()
  return sum + 10
end
```